### ISSUE TITLE
Documentos UML

### ISSUE THREAD

**Issue by lucasmendsc** [ifpe-cti/servicon-web#6]
@fabriciofx aonde eu devo colocar os diagramas UML,eles devem está no github? No readme.md talvez?

**Comment by fabriciofx** [comment]
@lucasmendsc Nope. Recomendo colocar uma pasta `docs` que esteja na raiz do projeto. Se não existir esta pasta, crie e coloque o diagrama lá. Depois você pode criar um arquivo Markdown que vai exibir este diagrama.
Detalhe:  em qual formato será este diagrama? 🤔

**Comment by lucasmendsc** [comment]
@fabriciofx o diagrama foi feito no astah,assim como os que ainda serão feitos. Esqueci de perguntar,qual era o site/programa que foi utilizado [neste](https://github.com/ifpe-cti/saudegaranhuns/blob/master/docs/uml/saudegaranhuns-classes.png) diagrama do saúde Garanhuns?
